src/main/resources/view/DarkTheme.css

```diff
@@ -17,6 +17,11 @@
     -fx-opacity: 1;
 }
 
+.title_label {
+    -fx-font-size: 20px;
+    -fx-text-fill: #FFFFFF;
```

Comment:
Perhaps you can consider using rgb values for your colour rather than hexadecimal numbering